internal/scraper/scraper_S_test.go

```diff
@@ -1329,13 +1330,15 @@ func TestScraper_S(t *testing.T) {
 						{Type: "HowToStep", Text: "Remove pot from heat. Stir in the chocolate, 1/2 tsp. salt, and the remaining 3 tbsp. vinegar. Serve warm, topped with garnishes, if using."},
 					},
 				},
-				Name:            "Veggie Chili",
-				NutritionSchema: &models.NutritionSchema{},
-				PrepTime:        "PT0H0M",
-				Tools:           &models.Tools{Values: []models.HowToItem{}},
-				TotalTime:       "PT0H0M",
-				Yield:           &models.Yield{Value: 8},
-				URL:             "https://www.sunset.com/recipe/veggie-chili",
+				Name: "Veggie Chili",
+				NutritionSchema: &models.NutritionSchema{
+					Calories: "0",
```

Comment:
Perhaps it would be better for the Calories to be `""` when 0 because not many recipes with multiple ingredients are empty on calories.

Reply:
That particular recipe returns the string "0" for the calories property, so that test case will fail if we are using an empty string.

After extracting the digits from the nutrition properties we do not clean any "0" values, which is the reason for that string value in the test case.

We could of course implement a clean-up of "0" values in the `EnsureNutritionUnitForString` function by setting such values to empty strings instead, which would make sure that these properties would show as "-" in the front end instead of e.g. "0 kcal".

How likely is it that the nutrition table will be converted from text to numbers in the database in the future?
Since that would be the most versatile solution, maybe it would be best to not implement too much logic to handle these edge cases as strings at this time.
@reaper47 What do you think?

Reply:
Good point! It is highly likely the nutrition properties in the `nutrition` table will be changed to `INTEGER` in the future because it will simplify things. The table might also benefit from `{property}_unit` columns. This being said, no action should be taken regarding "0" for now.

Reply:
Perfect. In that case this one is also resolved.